torch/ao/quantization/backend_config/executorch.py

```diff
@@ -47,6 +53,42 @@
     is_dynamic=True,
 )
 
+# xnnpack compatible dtype configs
+
+# We restrict scale values to be 2 ** -12 to ensure the
+# requantization scale never falls below the xnnpack lower
+# threshold. Additionally, for qint8 weight, we restrict
+# the quantization values to [-127, +127], excluding -128.
+# For more detail, refer to the description of
+# `default_symmetric_qnnpack_qconfig`.
+
+# TODO: add additional restriction on qscheme to ensure it
+# is either per_tensor_symmetric or per_channel_symmetric
+
+executorch_act_qint8_scale_min_2_neg_12 = DTypeWithConstraints(
+    dtype=torch.qint8,
+    scale_min_lower_bound=2 ** -12,
+)
+
+executorch_weight_qint8_neg_127_to_127_scale_min_2_neg_12 = DTypeWithConstraints(
```

Comment:
is there a difference between restrictions for executorch vs restrictions for qnnpack_running_xnnpack?  If yes, what are they?  If not, can we use the same objects?

I thought executorch was not specific to a device/backend, I'd imagine the restrictions like these should be for specific device/backend combination.  For example, xnnpack on edge, or a specific DSP chipset.

Reply:
it's the same, sure, I can import the same object from qnnpack.py